vehicle_api/models.py

```diff
@@ -0,0 +1,46 @@
+from sqlalchemy import Column, ForeignKey, Integer, String
+from sqlalchemy.orm import relationship
+from database import Base
+
+
+class Customer(Base):
+    __tablename__ = "customers"
+
+    id = Column(Integer, primary_key=True, index=True)
+    location = Column(String)
+    email = Column(String, unique=True, index=True)
+    phone = Column(String, unique=True)
+
+    deals = relationship(
+        "Deal", back_populates="customers", cascade="all, delete, delete-orphan", uselist="false"
+    )
+
+
+class Deal(Base):
+    __tablename__ = "deals"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String, index=True)
+    description = Column(String, index=True)
+
+    vehicles = relationship("Vehicle", back_populates="deals", cascade="all, delete, delete-orphan", uselist="false")
```

Comment:
I believe that there should be one vehicle per deal. Therefore it should be one-one relationship

Reply:
I searched how to add on-one relationship, and added **uselist="false"** for it, or is it wrong?

Reply:
It is correct. But using a plural form is not the best idea. I'd rename it to `vehicle`